packit/command_runner.py

```diff
@@ -0,0 +1,63 @@
+import logging
+from typing import Dict, Type
+
+from packit import utils
+from packit.config import RunCommandType
+from packit.local_project import LocalProject
+from generator.deploy_openshift_pod import OpenshiftDeployer
```

Comment:
with this line, packit will depend directly on python-kubernetes, do we want such a thing?

Reply:
We can import it in specific functions before calling them.
Another approach would be to load it from a specific directory over `load_module` https://www.programcreek.com/python/example/503/imp.load_module

Reply:
In this PR I need this dependency for running command into OpenShift. But I can import it here:
https://github.com/packit-service/packit/pull/364/files#diff-6dbb119e6096082f5622ab8413bc42d7R25

Otherwise, If there is another possibility, please let me know.

Reply:
+1 to import from within a function